lib/generator.rb

```diff
@@ -36,12 +37,24 @@ def test_cases
   end
 
   def generate
+    generate_test_file
+    update_version
+  end
+
+  private
+
+  def generate_test_file
     File.open(path_to("#{name.gsub(/[ -]/, '_')}_test.rb"), 'w') do |f|
       f.write ERB.new(File.read(path_to('example.tt'))).result binding
+      FileUtils.chmod('+x', f)
```

Comment:
I understand and can see that it would be a convenience.

If this is to be changed, I would think it should be a different pull request.

If the current pull request would need to be reverted for any reason, it would not effect pulling this change, which I consider to be unrelated.